src/records/RecordsConfig.cc

```diff
@@ -258,6 +258,10 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.udp.enable_gso", RECD_INT, "1", RECU_NULL, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.udp.max_recv_payload_size", RECD_INT, "65527", RECU_DYNAMIC, RR_NULL, RECC_NULL, "^[0-9]+$", RECA_NULL}
```

Comment:
We are going to have `proxy.config.quic.max_recv_udp_payload_size_in|out`, right?

Reply:
yes, my bad, made a mess with names and all that stuff. Will fix.